src/utilities/getSortedData.js

```diff
@@ -0,0 +1,15 @@
+const getSortedData = (sortDataList, sortBy) => {
+  if (sortBy === 'LOW_TO_HIGH') {
+    return sortDataList.sort((a, b) => {
```

Comment:
Since sort is not a pure function, it is a good practice to use it in the following manner
`[...sortDataList].sort()`

Reply:
thanks manish for suggestion will try to implement this.